app/routes.py

```diff
@@ -1,2 +1,96 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.planet import Planet
+from app import db
+
+planets_bp = Blueprint('planets', __name__, url_prefix='/planets')
+
+@planets_bp.route("", methods=['POST'])
+def create_planet():
+    request_body = request.get_json()
+    new_planet = Planet(
+        name = request_body["name"],
+        description = request_body["description"],
+        mass = request_body["mass"],
+    )
+
+    db.session.add(new_planet)
+    db.session.commit()
+
+    message = f"New planet {new_planet.name} successfully created, biatch!"
+
+    return make_response(message, 201)
+
+@planets_bp.route("", methods=["GET"])
+def get_planets():
+    mass = request.args.get("mass")
+
+    planets = Planet.query
+
+    if mass:
+        planets = planets.filter_by(mass=mass)
+
+    planets = planets.all()
+
+    request_body = []
+    for planet in planets:
+        request_body.append(
+            dict(
+                id=planet.id,
+                name=planet.name,
+                description=planet.description,
+                mass=planet.mass
+            )
+        )
+    return jsonify(request_body), 200
+
+def validate_planet(planet_id):
+    try:
+        planet_id = int(planet_id)
+    except:
+        abort(make_response({"message": f"Planet {planet_id} is invalid, sucker!"}, 400))
+    
+    planet = Planet.query.get(planet_id)
+
+    if not planet:
+        abort(make_response({"message": f"Planet {planet_id} was not found, sucker!"}, 404))
+
+    return planet
```

Comment:
Since validate_planet is a helper method, it is usually a good idea to move it either before or after the routes, or to its own file!